app/views/posts/show.html.slim

```diff
@@ -7,7 +7,7 @@
 
     .editor
       - if current_user && (@post.user && @post.user == current_user)
-        = react_component "PostPage", { post: PostSerializer.new(@post).serializable_hash, currentUser: current_user.present? ? UserSerializer.new(current_user) : nil }
+        = react_component "PostPage", { post: PostSerializer.new(@post).serializable_hash, currentUser: current_user.present? ? UserSerializer.new(current_user) : nil, isProcessing: @post.body.nil? }
```

Comment:
I think here `empty?` makes more sense?

Reply:
we can't use ```empty?``` because if ```post.body``` == ```nil``` we get an error

```empty?``` doesn't play nice with ```nil```

we can't use ```blank?``` because often reference posts have ```post.body``` == ```""``` which returns ```post.body.blank?``` == ```true```

other suggestions?

Reply:
what i am looking for is:

```post.body == nil``` returns ```true```
```post.body == ""``` returns ```false```
```post.body == "some text"``` returns ```false```

Reply:
sorry, I meant `blank?`, but :+1: